statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -308,6 +387,125 @@ def _reordered(self, order):
                       lag_order=self.k_ar, model=self.model,
                       trend='c', names=names_new, dates=self.dates)
 
+def orth_ma_rep(results, maxn=10, P=None):
+    r"""Compute Orthogonalized MA coefficient matrices using P matrix such
+    that :math:`\Sigma_u = PP^\prime`. P defaults to the Cholesky
+    decomposition of :math:`\Sigma_u`
+
+    Parameters
+    ----------
+    results : VARResults or VECMResults
+    maxn : int
+        Number of coefficient matrices to compute
+    P : ndarray (neqs x neqs), optional
+        Matrix such that Sigma_u = PP', defaults to the Cholesky decomposition.
+
+    Returns
+    -------
+    coefs : ndarray (maxn x neqs x neqs)
+    """
+    if P is None:
+        P = results._chol_sigma_u
+
+    ma_mats = results.ma_rep(maxn=maxn)
+    return mat([np.dot(coefs, P) for coefs in ma_mats])
+
+
+def test_normality(results, signif=0.05):
```

Comment:
Is this used anywhere other than inside `VARResults`?  If not, why not put the content in that method? (I have the same question about `forecast` in the status-quo branch)